src/main/java/com/group1/taskmanagement/config/CORSConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.group1.taskmanagement.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CORSConfig implements WebMvcConfigurer {
+
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+                .allowedOrigins("*")
+                .allowedMethods("GET", "POST", "PUT", "DELETE", "HEAD", "OPTIONS");
+    }
+}
```

Comment:
This whole file should not be here.

Reply:
It was deleted on main.

Reply:
The new CORS config moved to https://github.com/CMSC-495-Group-One/TaskManagementAPI/blob/1df2d2a1f3f526bef8362df8b823d825320a9617/src/main/java/com/group1/taskmanagement/security/SecurityConfig.java#L77C5-L87C6